content/en/apps/examples/covid-education.md

```diff
@@ -0,0 +1,49 @@
+
+## Solution Overview
+
+The CHT provides a flexible way to deliver timely COVID-19 remote training and education to CHWs. In collaboration with [Stanford's Digital Medic Team](https://digitalmedic.stanford.edu/), and vetted by a team of health experts at Stanford, a form-based application was designed to deliver three remote learning modules. Utilizing tasks to order the modules, CHWs can progress through training at their own pace. The training modules:
+
+* Build on verifiable health safety guidance from the CDC and WHO
+* Can be deployed using the CHT app or SMS workflows 
+* Capture assessment and understanding of CHW knowledge through quizzes, which can be tracked by Supervisors
+
+### Users and Hierarchy Example
+
+| User                             | Location                               | Devices                             | Role                                                                                                                                                                                                                                              |
+| :------------------------------- | :------------------------------------- | :---------------------------------- | :------------------------------------------------------------------------------------------------- |
+| National Officials and County Teams  | Central and district offices           | Desktop, laptop and smartphones     | Monitor central analytics regarding onboarding and training completion. Revisit content where needed.           |
+| Sub-County Teams       | Facilities or local offices            | Tablet or smartphone                | Review CHW progress on aggregate. Support CHW supervisors and monitor location specific analytics.            |
+| CHW Supervisors                  | Community level, based at facilities   | Smartphones and personal phones     | Verify CHW onboarding and training completion. Follow-up with those who have not completed training. Complete their own onboarding and training tasks.                                    |
+| CHWs and Volunteers              | Community level                        | Feature phones and some smartphones | Complete onboarding and training via SMS or App. Provide feedback and ask for support where needed.         |
+
+## Workflow Example
+
+This demo illustrates how a CHT workflow can be easily adapted to integrate a sequence of learning modules through the usage of tasks. These modules are designed to be delivered sequentially to promote engagement and absorption of content. Learning can be tracked across CHWs to inform opportunities to provide additional support. The sequence can also be modified to interact with the CHW through SMS, utilizing 'question and reply' text-based workflows.
```

Comment:
A bit of a personal preference, but the word "utilizing" is less approachable to readers.

```suggestion
This demo illustrates how a CHT workflow can be easily adapted to integrate a sequence of learning modules through the usage of tasks. These modules are designed to be delivered sequentially to promote engagement and absorption of content. Learning can be tracked across CHWs to inform opportunities to provide additional support. The sequence can also be modified to interact with the CHW through SMS, using question-and-answer text-based workflows.
```